README.rst

```diff
@@ -75,9 +75,12 @@ https://github.com/diffpy/diffpy.utils.
 
 Feel free to fork the project and contribute.  To install diffpy.utils
 in a development mode, with its sources being directly used by Python
-rather than copied to a package directory, use ::
+rather than copied to a package directory, use the following in the root
+directory ::
+
+   pip install -e .
 
-   python setup.py develop --user
+Note that this is only supported for `setuptools` version 62.0 and above.
```

Comment:
instead of this, do we just want  to pin setuptools in the requirements to be `>=62.0`?